Makefile

```diff
@@ -23,6 +23,10 @@ release-build:
 		--sources-content=false
 .PHONY: release-build
 
+view-build:
+	npm run --prefix astexplorer build
```

Comment:
Could you switch to `yarn` - we use it elsewhere in the file.

Reply:
Done in fdec035cf52bfbf50580da4b1ef9711eb838f40d